v63004/u_inref/gtm8791.exp

```diff
@@ -13,20 +13,26 @@
 set timeout 60
 spawn /usr/local/bin/tcsh -f
 
+expect_after {
+	timeout { timeout_procedure }
+}
+
+proc timeout_procedure { } {
+	puts "timeout occurred"
+	exit -1
+}
+
+
 send -- "\$gtm_dist/lke\r"
 
-expect -exact "LKE>"
 # Send <Ctrl-Z>
-send -- "\x1A\r"
+expect -exact "LKE>" {send -- "\x1A\r"}
```

Comment:
Now that we will exit the expect script the moment a timeout happens, do we need the change in this line? Seems like we can have expect commands and send commands in separate lines like before.

Reply:
If thats preferable I'll go ahead and change the send lines back to the former syntax.